ref/feature/spnego/examples.adoc

```diff
@@ -0,0 +1,27 @@
+== Examples
+
+Open Liberty supports many common SPNEGO scenarios and configurations.
+
+=== Limit SPNEGO authentication 
+You can filter HTTP requests to require SPNEGO authentication for only certain requests, web applications, hosts, or user agents.
+
+The following example configures the following things:
+<1> SPNEGO authentication is used for any requests that are received containing the `example.com` hostname for resources within the `protectedApp` web application.
+<2> The client's GSS credentials are not added to the user subject upon successful authentication.
+<3> The Kerberos configuration and keytab files to be used by the server are given specific locations within the server configuration directory instead of their respective default locations.
```

Comment:
'keytab files' should there be backticks for `keytab`?

Reply:
From what I was able to find, I don't think it does.